addons/cluster-autoscaler/v1.6.0.yaml

```diff
@@ -189,6 +189,7 @@ spec:
         - name: ssl-certs
           hostPath:
             path: {{SSL_CERT_PATH}}
+      dnsPolicy: "Default"
```

Comment:
Any way to make this host only, and ignore kube dns?

Reply:
The kubernetes docs are a bit lacking here, from my understanding, `dnsPolicy: Default` is exacty that, setting the dns lookup to be host only. The default value for dnsPolicy is surprisingly not Default, it's `ClusterFirst` which is kube dns only 😛